doc/classes/CanvasItem.xml

```diff
@@ -622,7 +623,7 @@
 			The rendering layer in which this [CanvasItem] is rendered by [Viewport] nodes. A [Viewport] will render a [CanvasItem] if it and all its parents share a layer with the [Viewport]'s canvas cull mask.
 		</member>
 		<member name="visible" type="bool" setter="set_visible" getter="is_visible" default="true">
-			If [code]true[/code], this [CanvasItem] is drawn. The node is only visible if all of its ancestors are visible as well (in other words, [method is_visible_in_tree] must return [code]true[/code]).
+			If [code]true[/code], this [CanvasItem] may be drawn. Whether this [CanvasItem] is actually drawn depends on the visibility of all of its [CanvasItem] ancestors. In other words: this [CanvasItem] will be drawn when [method is_visible_in_tree] returns [code]true[/code] and all [CanvasItem] ancestors share at least one [member visibility_layer] with this [CanvasItem].
```

Comment:
The second sentence is not fully true. Not necessarily all CanvasItem ancestors matter, as non-CanvasItems break the propagation chain.
![vMN5Ndwtzv](https://github.com/user-attachments/assets/8be3a31a-7d68-4405-9735-52d3339a066d)

(I guess I can keep coming with counterexamples on and on. :stuck_out_tongue:)

Also it still doesn't mention CanvasLayer/Window. :thinking:

Reply:
> I did try to also explain the stuff that's already in `is_visible_in_tree` also in `visible`, but it got reeeeaaaalllly long and I don't think anyone wants to real long descriptions of anything. I don't disagree with you though; just didn't find a 'nice' way to condense all that info here.

Yeah I absolutely get it, that's why I ended my first comment here (https://github.com/godotengine/godot/pull/97223#discussion_r1770570180) with:

>Just saying, no idea if/how exactly you want to clarify all of that in here.

The _"if"_ meant to imply that I'd understand if we don't really want to be too elaborate on that.

But OTOH if someone is trying to clarify the docs to be more detailed / technically correct... it's hard to resist pointing out they're still not really correct. Like it's kinda easier to ignore when the docs are completely vague than when they fake to be detailed. :upside_down_face: